README.md

```diff
@@ -6,19 +6,19 @@ Lead Maintainer: [Matt Edelman](https://github.com/grawk)
 
 Identify all locales under a given directory and call a passed in "writer" for each one
 
-## API
-
-`module.exports = function build(appRoot, writer, cb) {`
+## Options
 
 - `appRoot {String}` filesystem directory where `locales` directory resides. Under that would be structure e.g. `US/en`, `XC/zh`
-- `writer {Function}`
-  - `localeRoot {String}` root output directory for the given locale
+- `buildPath {String}` the directory relative to `appRoot` where compiled files will be placed, default: `.build`
+- `localesPath {String}` the directory relative to `appRoot` where locale files can be found, default: `locales`
```

Comment:
Same as above: what's the reason for this? (Justifying changes is super useful when reviewing, so there's a why to evaluate)